search/test_utils.py

```diff
@@ -30,3 +30,23 @@ def test_delete_unexisting_indicator_raises_exception(self):
         indexer.delete_indicator(indicator)
 
         self.assertRaises(ValueNotFoundError, indexer.delete_indicator, indicator)
+
+    def test_create_and_delete_workflowlevel1(self):
+        indexer = ElasticsearchIndexer()
+        org = Organization.objects.create(organization_uuid="index-workflowlevel1-test")
+        wflvl1 = factories.WorkflowLevel1(organization=org)
+        indexer.delete_workflowlevel1(wflvl1)
```

Comment:
Just calling the function checks that the function does not crash, but not that it actually deletes the record. Can you make a simple check of that?

Reply:
Well i might do that. But it is not really a change since both tests would rely entirely on ElasticSearch to run correctly. So the assumption we have to set (That ES is running correctly) is the same for both cases.